telemetry_models/metrics/overview.yaml

```diff
@@ -0,0 +1,95 @@
+description: >
+  Metrics are numeric measurements of some aspect. In observability it is often a 
+  measurement of a system or applications behavior. Many metrics are summaries of 
+  or distillations of behavior over a small period of time. 
+  
+attributes:
+  - attribute: identifier
+    description: >
+      A means of uniquely identifying the measurement amongst a group of measurements 
+      from other systems, sources or measurements. 
+      
+      Earlier identifiers consist of a single string that is often partitioned by a 
+      character such as the period to create a hierarchy of data. E.g. 
+      `sys.cpu.idle.percent.us-east-1.i-1234`
+      
+      Later identifiers consisted of a metric name with a set of key/value tags or 
+      dimensions. For example `sys.cpu.idle.percent region=us-east-1 instance=i-1234`.
+      Variations on dimensional identifiers include merging the metric name with
+      the dimensions, multi-value dimension values and key-only dimensions.
+
+  - attribute: timestamp
+    description: >
+      A point in time associated with the measurement. This may be the time when the
+      measurement was taken or or reported. It may also be the start or end of an
+      interval the measurement represents.
+
+  - attribute: value
+    description: >
+      Most commonly a single numeric value, particularly from instrumentation sources.
+      Some systems support integer values, others IEEE 754 single or double precision
+      floating point values. Some systems include multiple numeric values as summaries
+      for an interval. E.g. sum, min, max, count, avg, etc. Others may include special
+      mer-gable structures for estimation of quantiles or cardinalities.
+
+uses:
+  - title: Control Plane
+    description: >
+      Since metrics are well defined and easily machine readable, metrics are often
+      used in control planes to perform automated tasks such as scaling resources
+      or distributing work.
+
+  - title: Alerting
+    description: >
+      Again, since they are well defined and easily machine readable, metrics are 
```

Comment:
If you expect readers may jump around, then `Again` may not be appropriate as they may not have read the previous bullet.

Reply:
good point.